src/js/app.js

```diff
@@ -1,6 +1,3 @@
-// Import Sass
-import "../styles/app";
```

Comment:
You can load source style files directly in template.

Of cause you can import styles in JS  too, but then you should use the `.scss` extension:
```js
import "../styles/app.scss";
```